src/views/2_SessionComponent.tsx

```diff
@@ -25,6 +25,10 @@ export class SessionComponent extends React.Component<Props, {}> {
     }
 
     componentDidMount() {
+        this.session.onscroll = () => {
+            const shutterOffset = this.session.scrollTop;
+            this.shutter.style.transform = `translateY(${shutterOffset}px)`;
+        };
```

Comment:
I tried that for a long time, I'm pretty sure it's not possible with CSS only. The issue is that if the shutter div is on top of the content div, it captures the scroll events, and the content div can't be scrolled. There is no way to force the shutter div to be the same size as the content either. Stack Overflow couldn't come up with anything either: http://stackoverflow.com/questions/38760571/cover-a-div-with-a-transparent-overlay